tests/test_ultrahub.py

```
@@ -59,12 +56,10 @@ async def _request(*_args: object, **_kwargs: object) -> FakeResponse:
     api = VodafoneStationUltraHubApi(
         base_url, "u", "p", cast("Any", FakeSession(request_impl=_request))
     )
-    reply_json, response = cast(
-        "tuple[dict[str, Any], FakeResponse]",
-        asyncio.run(_acall(api, "_auto_hub_request_page_result", HTTPMethod.GET, "x")),
+    reply_json = asyncio.run(
+        _acall(api, "_auto_hub_request_page_result", HTTPMethod.GET, "x")
     )
     assert reply_json == {"csrf_token": "t"}
-    assert response.status == HTTP_OK
     assert api.csrf_token == "t"
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to: "csrf_token"

(S105)